### PR TITLE
Map hydro storage for AR

### LINKED ISSUE #1722: Pumped storage generation in Argentina
There are two hydro pumped storage power plants in Argentina!
We're currently mapping them as usual 'hydro'
https://en.wikipedia.org/wiki/Los_Reyunos_Dam
https://en.wikipedia.org/wiki/Rio_Grande_Hydroelectric_Complex

They have the ending "HB" in the list of generators:
http://portalweb.cammesa.com/memnet1/revistas/estacional/base_gen.html
image

I suggest pushing them to 'hydro storage'.

### PR DESCRIPTION
**Pull request by systemcatch**
fixes #1722 

Also made a few small tweaks, one stops the warning about plants we know to be in Uruguay, the other makes sure a float is returned for zero generation of a type.

### CODE REVIEW

File: parsers/AR.py [hunk 1]
```diff
@@ -825,6 +833,13 @@ def fetch_production(zone_key='AR', session=None, target_datetime=None, logger=l
     thermal = get_thermal(session, logger)
     hydro = get_hydro_and_renewables(session, logger)
 
+    # discharging is given positive value in data, opposite to EM
+    hydro_storage = hydro.pop('hydro_storage')
```
[the remark this answers is not in the document]
Comment (systemcatch) [reply parsers/AR.py]: Is receiving -0.0 not a problem then?

**Comment by corradio** [comment]
Does -0.0 >= 0 in python? I have doubts now and am afk so can't test

**Comment by systemcatch** [comment]
Quickly checked it.
```shell
Python 3.5.2 (default, Nov 12 2018, 13:43:14) 
[GCC 5.4.0 20160609] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> -0.0==0.0
True
>>> -0.0>=0.0
True
```

File: parsers/AR.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (corradio) [reply parsers/AR.py]: it seems not to be

**Comment by corradio** [comment]
both are fine with me

**Comment by alixunderplatz** [comment]
It's good that the dubious datapoints are excluded for now. Unknown had about 1.3-2.2 GW due to these strange entries in the table.

I'll leave another comment on the whole UY-AY-hydro-thing in issue #1633

**Pull request merged**